package proto_ff;

import "yd_fieldoptions.proto";


message backback
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "唯一id"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "找回类型"];
	optional int32 typeParam = 3[(yd_fieldoptions.field_cname) = "类型参数"];
	optional string name = 4[(yd_fieldoptions.field_cname) = "显示名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 group = 5[(yd_fieldoptions.field_cname) = "组"];
	optional int32 functionUnlock = 6[(yd_fieldoptions.field_cname) = "关联功能开放"];
	optional int32 levelLimit = 7[(yd_fieldoptions.field_cname) = "等级要求下限"];
	optional int64 PboxId = 8[(yd_fieldoptions.field_cname) = "完美找回物品box"];
	optional int64 boxId = 9[(yd_fieldoptions.field_cname) = "找回物品box"];
	optional int32 backDay = 10[(yd_fieldoptions.field_cname) = "找回天数"];
	optional int32 costItemID = 11[(yd_fieldoptions.field_cname) = "普通找回消耗物"];
	optional int32 costNum = 12[(yd_fieldoptions.field_cname) = "普通找回消耗数量"];
	optional int32 PcostItemID = 13[(yd_fieldoptions.field_cname) = "完美找回消耗物"];
	optional int32 PcostNum = 14[(yd_fieldoptions.field_cname) = "完美找回消耗数量"];
	optional int32 AcostItemID = 15[(yd_fieldoptions.field_cname) = "额外找回消耗物"];
	optional int32 AcostNum = 16[(yd_fieldoptions.field_cname) = "额外找回小号数量"];
	repeated int32 param = 17[(yd_fieldoptions.field_cname) = "参数", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_backback
{
	repeated backback backback_List = 1[(yd_fieldoptions.field_arysize)=20];
}
